src/components/modals/Send/fields/RecipientField.js

```diff
@@ -97,7 +101,7 @@ class RecipientField<Transaction> extends Component<
     const error =
       !value || isValid
         ? QRCodeRefusedReason
-        : new InvalidAddress(null, { currencyName: account.currency.name })
+        : warning || new InvalidAddress(null, { currencyName: account.currency.name })
```

Comment:
So if I understand correctly we for now consider warning as error as workaround for the fact we don't have yet the better bridge we did on mobile? That's a smart workaround, we'll refactor when sharing more code soon for bridge.


There are legit warning like in case of ethereum when the address is in lowercase and it displays as yellow if Im correct. Do you confirm this doesn't break this case?

Reply:
Ethereum warning for lowercase (the lacking of a checksum) shows in yellow since it still passes the `isValid`, so there's no error. In the workaround for XRP it will not be valid so it will be in red, 

![image](https://user-images.githubusercontent.com/4631227/52263157-b09fae00-292e-11e9-8879-b9fb8bd86b0a.png)

![image](https://user-images.githubusercontent.com/4631227/52263230-e17fe300-292e-11e9-92a2-552bd867dbd3.png)

As always I'll feel happier after arnaud goes over it :trollface: